presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -123,4 +124,17 @@ public GlueHiveMetastoreConfig setAwsSecretKey(String awsSecretKey)
         this.awsSecretKey = Optional.ofNullable(awsSecretKey);
         return this;
     }
+
+    public Optional<String> getCatalogId()
+    {
+        return catalogId;
+    }
+
+    @Config("hive.metastore.glue.catalogid")
```

Comment:
You could call this parameter `hive.metastore.glue.catalog-id` (added hyphen) for consistency with the other "multi-word" options? WDYT?

Reply:
Sorry. I skipped over your comment. My bad.

Reply:
No worries.  I wouldn't mind making this consistent with the other setting names, but it may cause some confusion for users migrating from EMR.  Maybe it would be ok if it was documented as a difference between the two implementations.

Reply:
We could actually support both:
```
@LegacyConfig("hive.metastore.glue.catalogid")
@Config("hive.metastore.glue.catalog-id")
```